### ISSUE TITLE
Options on aggregation doesn't work (Mongo 2.6rc specific)

### ISSUE THREAD

**Issue by jugglingcats** [paralect/robomongo#495]
Mongo 2.6rc has options as second argument to aggregate command. If you enter these in robomongo shell the aggregation is executed but the options are ignored.

Eg.

db.audit.aggregate([
    {
        $match: {
            "meter.count": { $gte: 0 }
        }
    }
    ...... more stages .....
],
{ 
    explain: true
}
)

Thanks for a great tool!

**Comment by ant-fx** [comment]
Any update on this? Would be nice to see explain reports from within Robomongo

**Comment by stennie** [comment]
@ant-fx As noted in a previous comment on this issue, you can currently use the `runCommand()` syntax to pass the explain option to the `aggregate` command. Note that the aggregation explain information (as at MongoDB 2.6) shows how the pipeline will be executed, but has less detailed stats than running an explain on the equivalent `find()` query. The MongoDB manual has some helpful information on [aggregation pipeline optimization](http://docs.mongodb.org/manual/core/aggregation-pipeline-optimization/).

The `aggregate()` helper will be updated when #520 is resolved (tagged for a 0.9.0 milestone). I recently started working on merging the 2.8 shell (as at 2.8.0-rc4) but that's still a sizeable effort.

**Comment by anorsich** [comment]
@jugglingcats @rfink Latest [0.9.0](http://blog.robomongo.org/robomongo-0-9-0-rc3-released/) is based on Mongodb shell 3.2 and support aggregate arguments. 
Here is sample respose with `explain: true`:

```
/* 1 */
{
    "stages" : [ 
        {
            "$cursor" : {
                "query" : {
                    "test" : false
                },
                "plan" : {
                    "cursor" : "BasicCursor",
                    "isMultiKey" : false,
                    "scanAndOrder" : false,
                    "allPlans" : [ 
                        {
                            "cursor" : "BasicCursor"
                        }
                    ]
                }
            }
        }
    ],
    "ok" : 1
}
```
